src/main/java/org/jenkinsci/plugins/workflow/cps/CpsWhitelist.java

```diff
@@ -103,6 +104,9 @@ public boolean permitsStaticMethod(Method method, Object[] args) {
             String driverTo = "org.codehaus.groovy.runtime.";
             if (cn.startsWith(driverFrom)) {
                 try {
+                    if (Modifier.isPrivate(method.getModifiers())) {
+                        return true;
```

Comment:
So the above is what I'm trying and the test is failing, 'cos the `ProxyEnv` subtype is cascading further than desired. I will now figure out how to stop that. =)